package.json

```diff
@@ -40,7 +40,8 @@
     "mustache": "^2.2.1",
     "mutationobserver-shim": "^0.3.1",
     "pbf": "^1.3.2",
-    "point-in-polygon": "1.0.0",
+    "point-geometry": "^0.0.0",
+    "point-in-polygon": "steve9164/point-in-polygon",
```

Comment:
Also, if all you need is something with an `x` and `y` property to represent a point, you can use Cesium's `Cartesian2` instead of pulling in `point-geometry`.